pkgs/applications/science/biology/cnvkit/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchPypi
+, rPackages
+, rWrapper
+, buildPythonPackage
+, six
+, biopython
+, numpy
+, scipy
+, pandas
+, matplotlib
+, reportlab
+, pysam
+, future
+, pillow
+, pomegranate
+}:
+
+let
+  pyfaidx = buildPythonPackage rec {
```

Comment:
I would move this somewhere else so that it can be overriden/patched.

Reply:
Moved to new python development module.